### PR TITLE
WIP: Add galaxy2cwl to Dockerfile

### PR DESCRIPTION
**Pull request by stain**
Adds `galaxy2cwl` on path, but is not yet called by the galaxy parser in Ruby

### CODE REVIEW

**Comment by stain** [comment]
I also added `cwltool`. Perhaps *TODO* is to move the `pip install` arguments out to a `requirements.txt` that can be installed also outside Docker?

**Comment by fbacall** [comment]
> I also added `cwltool`. Perhaps _TODO_ is to move the `pip install` arguments out to a `requirements.txt` that can be installed also outside Docker?
:+1: to this. We have quite a few non-container deployments

**Comment by stuzart** [comment]
it should also go in the install docs and the upgrade notes

**Comment by stuzart** [comment]
I'll be adding the requirements.txt as part of getting the travis working correctly, it seems to be the recommended way

File: Dockerfile [hunk 1]
```diff
@@ -13,18 +13,23 @@ RUN apt-get update -qq && \
...
 RUN mkdir -p $APP_DIR
-RUN chown www-data $APP_DIR
+RUN chown -R www-data $APP_DIR /var/www
 
 USER www-data
 
 WORKDIR $APP_DIR
 
+# CWL dependencies, galaxy2cwl
+RUN pip3 install cwltool==3.0.20200324120055 html5lib==1.0.1 galaxy2cwl==0.1.4
```
Comment (stuzart) [inline Dockerfile:30]: this could use the requirements.txt which contains cwltool, html5lib and galaxy2cwl

**Pull request merged**